modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -63,7 +63,26 @@ spec:
 . *Deploy the Operator.* Use the files that were generated into the `deploy/`
 directory by the Operator SDK when you built the Memcached Operator.
 
-.. Apply the Operator's CSV manifest to the specified namespace in the cluster:
+.. Edit the generated CSV manifest file by adding `description` and `displayName` fields for each custom resource definition (CRD) `kind` in the `spec.customresourcedefinitions.owned` section:
++
+.deploy/olm-catalog/memcached-operator/0.0.1/memcached-operator.v0.0.1.clusterserviceversion.yaml file
+[source,yaml]
+----
+...
+spec:
+  customresourcedefinitions:
+    owned:
+    - kind: Memcached
+      name: memcacheds.cache.example.com
+      version: v1alpha1
+      description: Placeholder description <1>
```

Comment:
Maybe something besides "Placeholder description" here? 
Maybe `<crd_description>` or something?

Reply:
Yeah, was deciding between that or what the other similar autogenerated fields in the file use (the "Placeholder description"). Think I'm flip back to an actual replaceable.

Reply:
Sounds good. When I saw "placeholder" my first thought was that it was an internal note.